### PR TITLE
update bootstrap guide

### PR DESCRIPTION
**Pull request by ozkanonur**
Coupled with https://github.com/rust-lang/rust/pull/107116

### CODE REVIEW

File: src/building/bootstrapping.md [hunk 1]
```diff
@@ -135,31 +135,6 @@ bootstrapping the compiler.
 [intrinsics]: ../appendix/glossary.md#intrinsic
 [ocaml-compiler]: https://github.com/rust-lang/rust/tree/ef75860a0a72f79f97216f8aaa5b388d98da6480/src/boot
 
-## Contributing to bootstrap
```
Comment (jyn514) [inline src/building/bootstrapping.md:138]: Maybe link to src/bootstrap/README somewhere? At the very beginning with a warning that it's internal documentation, maybe?

**Review by jyn514 — APPROVED** [review]

File: src/building/bootstrapping.md (hunk 1 above)
Comment (ozkanonur) [reply src/building/bootstrapping.md:138]: Yeah, good idea

File: src/building/bootstrapping.md [hunk 2]
```diff
@@ -16,6 +15,11 @@ version.
 This is exactly how `x.py` works: it downloads the current beta release of
 rustc, then uses it to compile the new compiler.
 
+Note that this documentation mostly covers user-face informations. See
```
Comment (jyn514) [inline src/building/bootstrapping.md:18]:
```suggestion
Note that this documentation mostly covers user-facing information. See
```

File: src/building/bootstrapping.md [hunk 3]
```diff
@@ -16,6 +15,11 @@ version.
...
+[bootstrap/README.md][bootstrap-internals] to read bootstrap internals.
```
Comment (jyn514) [inline src/building/bootstrapping.md:19]:
```suggestion
[bootstrap/README.md][bootstrap-internals] to read about bootstrap internals.
```

**Review by jyn514 — APPROVED** [review]
Thanks! Looks great with the typos fixed :)

**Pull request merged**